src/hooks/spacewalk/useBridgeSettings.ts

```diff
@@ -29,17 +30,21 @@ function useBridgeSettings(): BridgeSettings {
 
   useEffect(() => {
     const combinedVaults: ExtendedRegistryVault[] = [];
-    Promise.all([getVaultsWithIssuableTokens(), getVaultsWithRedeemableTokens()]).then((data) => {
-      getVaults().forEach((vaultFromRegistry: any) => {
-        const vaultWithIssuable = data[0]?.find(([id, _]) => id.eq(vaultFromRegistry.id));
-        const vaultWithRedeemable = data[1]?.find(([id, _]) => id.eq(vaultFromRegistry.id));
-        const extended: ExtendedRegistryVault = vaultFromRegistry;
-        extended.issuableTokens = vaultWithIssuable ? vaultWithIssuable[1] : undefined;
-        extended.redeemableTokens = vaultWithRedeemable ? vaultWithRedeemable[1] : undefined;
-        combinedVaults.push(extended);
+    Promise.all([getVaultsWithIssuableTokens(), getVaultsWithRedeemableTokens()])
+      .then((data) => {
+        getVaults().forEach((vaultFromRegistry: any) => {
+          const vaultWithIssuable = data[0]?.find(([id, _]) => id.eq(vaultFromRegistry.id));
+          const vaultWithRedeemable = data[1]?.find(([id, _]) => id.eq(vaultFromRegistry.id));
+          const extended: ExtendedRegistryVault = vaultFromRegistry;
+          extended.issuableTokens = vaultWithIssuable ? vaultWithIssuable[1] : undefined;
+          extended.redeemableTokens = vaultWithRedeemable ? vaultWithRedeemable[1] : undefined;
+          combinedVaults.push(extended);
+        });
+        setExtendedVaults(combinedVaults);
+      })
+      .catch(() => {
+        toast('Could not establish connection with the bridge...', { type: toast.TYPE.ERROR, toastId: 2138 });
```

Comment:
What's that `toastId` doing?

Reply:
Ah, I see 👍 but the ID was chosen arbitrarily?

Reply:
Hmm yes, let's try that. It's not super necessary but makes it clearer what they are used for and how they are assigned. And it's easier to check which numbers are already assigned 👍

Reply:
@ebma I've added a `showToast.ts` to create an abstraction over toast to make the toast management cleaner. I implemented it for toast messages without any variables. Do you think I should implement it also for toast messages with variables?

Reply:
Hmm yes, I think it's better to streamline it, might cause confusion otherwise. How about we just extend your new function with an optional parameter `customMessage?: string` and if this `customMessage` is passed to the function you ignore the `message` contained in `ToastProperties` but rather use that. This way, the caller can still use variables for the message but we don't have to create some complex string replacement logic.